### DISCUSSION TITLE
Create deploy hook programmatically?

### DISCUSSION THREAD

**Discussion by EmilePW** [vercel/vercel#5953] [Ideas]
I'd like to be able to create deploy hooks using the API (or any other method than I run in code rather than having to it via the UI). Are there any plans for this?

**Comment by martinkz** [comment]
I'm also looking for a way to do this. We want to deploy dozens of marketing pages as their own projects on a weekly basis and ideally not have to manually set up deploy hooks for each one via the dashboard. Would be nice if it was possible via the vercel CLI. 

Any ideas for a workaround?
